tests/test_config.py

```diff
@@ -120,13 +120,13 @@ def test_init(self):
         '11154', 'Test attribute users 1',
         '["and", ["or", ["or", {"name": "test_attribute", "type": "custom_attribute", "value": "test_value_1"}]]]',
         conditionStructure=['and', ['or', ['or', 0]]],
-        conditionList=[['test_attribute', 'test_value_1']]
+        conditionList=[['test_attribute', 'test_value_1', 'custom_attribute', None]]
```

Comment:
We haven't updated `project_config.py` code. How is this getting set to None?

Reply:
We have edited `def _audience_condition_deserializer(obj_dict):` to add type and match to attribute node so that we use it in `CustomAttributeConditionEvaluator - evaluate()`
https://github.com/optimizely/python-sdk/pull/146/files#diff-54739e7fb9eb3054932a2f39734e8676R218-R232